docs/API.md

```diff
@@ -369,6 +369,13 @@ parameters will be reduced accodringly.
 Flushes the content of the buffer in [extreme mode](extreme.md). It has no effect if
 extreme mode is not enabled.
 
+<a id="flushSync"></a>
+## .flushSync()
+
+### Discussion:
```

Comment:
I don't believe "Discussion:" is a common idiom in the documentation - can you please remove this for consistency

Reply:
I'm confused... "Discussion" is all over the place. I just copy pasted the `flush` section and slightly modified it.

Reply:
@davidmarkclements it's the only thing I could think to title the subsection for each part of the API.